bibtasklets/bst_fermilab.py

```diff
@@ -106,10 +109,14 @@ def bst_fermilab():
             if report[4]:
                 search2 = '035__a:' + report[4]
                 result = perform_request_search(p=search2, cc='HepNames')
-                report[2] = '<a href="%(site)s/record/%(recid)s">%(rep2)s</a>'\
-                            % {'site': CFG_SITE_URL,
-                               'recid': str(result[0]),
-                               'rep2': report[2]}
+                try:
+                    report[2] = '<a href="%(site)s/record/%(recid)s">%(rep2)s</a>'\
+                                % {'site': CFG_SITE_URL,
+                                   'recid': str(result[0]),
+                                   'rep2': report[2]}
+                except:
```

Comment:
a naked `except` is generally frowned upon
you want to list the type of Exceptions that you expect the code could throw,
e.g. `IndexError` or `TypeError` ...

anyhow, in this case, why not simply `if result`   `else`  .... 
ERRORS.append("INSPIRE-ID:xxx not found in HepNames")